espanso-detect/src/mac/mod.rs

```diff
@@ -403,7 +403,7 @@ fn key_code_to_key(key_code: i32) -> (Key, Option<Variant>) {
         // Modifiers
         0x3A => (Alt, Some(Left)),
         0x3D => (Alt, Some(Right)),
-        0x39 => (CapsLock, None), // TODO
```

Comment:
It's impossible to know what lonely `TODO` comments intentions were, so we delete them